packages/preview/light-cv/0.1.0/template.typ

```diff
@@ -0,0 +1,157 @@
+#import "@preview/fontawesome:0.1.0": *
+
+#import "settings/styles.typ": *
+
+#import "modules/utils.typ": *
+#import "modules/header.typ": *
+#import "modules/section.typ": *
+#import "modules/skills.typ": *
```

Comment:
Normally, the files you stored within the `modules` dictionary are not copied into the user project. Instead, they are imported through the package manager.

I would suggest that you move your settings folder and `cv-en.typ` to a new folder `template` and make that directory the template's path in the `typst.toml`. Then, you can create a new `lib.typ` and reexport all relevant functions from the `template.typ` and the files in `modules` there and make it your entrypoint.

For orientation, I have attached a screenshot of the directory structure your template generates when used with `typst init` and how it looks like for other templates below: 

![Finder screenshot of the directory structure of `starter-journal-article` and `light-cv`](https://github.com/typst/packages/assets/3874949/65eb6095-dcbd-4fbf-963f-53c25b12a3b3)